server.js

```diff
@@ -1,39 +0,0 @@
-const webpackDevMiddleware = require('webpack-dev-middleware');
```

Comment:
Please decide if this is an issue for you. In my opinion the less requirements we have, the less configuration we use, the more chances are that someone will just take this example and use as a base for his project (I would)